bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -250,6 +262,29 @@ public String getDataUrl() {
         }
     }
 
+    @Override
+    @JsonIgnore
+    public String getCustomFunctionUrl() {
+        String customFunctionUrl = "";
```

Comment:
are you we using any feature toggles in public code nowadays? cc @rismehta

Reply:
Using FT on public code is not recommended. We don't use it yet anywhere, if there is a use-case, the design needs to be re-visited.